### PR TITLE
Counting: Support delimiters

### PR DESCRIPTION
**Pull request by winstonsung**
- Support 4-digit whitespace delimiter
- Support 3-digit comma delimiter

Bug: #34
Bug: #52
Change-Id: I377ce642e5cffbbdd08ca681d39ac22d9ecc06ee

### CODE REVIEW

File: cog/comment.py [hunk 1]
```diff
@@ -109,9 +110,43 @@ def today_comment(userId, message, CURSOR):
...
+            # - "0 b 0000"
+            # - "0 b 0 0000"
+            if re.match(
+                "^(0[bdox]|0[bdox] |0 [bdox] |)" +
+                    "([0-9A-Fa-f]{1,4})" +
+                    "(([0-9A-Fa-f]{4})*|( [0-9A-Fa-f]{4})*)$",
+                based_number
+            ):
+                based_number.replace(" ", "")
+            # If is valid 3-digit comma delimeter format
```
Comment (YuevUwU) [inline cog/comment.py:141]: `str.replace`不直接修改`str`的，應改為
```py
based_number = based_number.replace(" ", "")
```

File: cog/comment.py [hunk 2]
```diff
@@ -109,9 +110,43 @@ def today_comment(userId, message, CURSOR):
...
+            # (10-based, without base)
+            elif (
+                counting_base == 10 and
+                re.match("^([0-9]{1,3}(,[0-9]{3})*)$", based_number)
+            ):
+                based_number.replace(",", "")
```
Comment (YuevUwU) [inline cog/comment.py:147]: `str.replace`不直接修改`str`的，應改為
```py
based_number = based_number.replace(",", "")
```

File: cog/comment.py (hunk 1 above)
Comment (winstonsung) [reply cog/comment.py:141]: Thanks for the reminder. Should be fixed now.

File: cog/comment.py (hunk 2 above)
Comment (winstonsung) [reply cog/comment.py]: Thanks for the reminder. Should be fixed now.

**Pull request merged**